### ISSUE TITLE
Feature request: pixelated images

### ISSUE THREAD

**Issue by MarkJeronimus** [Jezzamonn/fourier#7]
As one who studied signal processing, I disagree with the way the JPEG wavelets and the cat image are displayed. They shouldn't be displayed resampled but pixelated because only the values at integer coordinates matter.

This can be achieved with these extra CSS rules:
```CSS
        image-rendering: -moz-crisp-edges;         /* Firefox */
        image-rendering:   -o-crisp-edges;         /* Opera */
        image-rendering: -webkit-optimize-contrast;/* Webkit (non-standard naming) */
        image-rendering: crisp-edges;
        -ms-interpolation-mode: nearest-neighbor;  /* IE (non-standard property) */
```

![Image](https://user-images.githubusercontent.com/10319630/56086140-144dc800-5e05-11e9-9b20-ea96a1627f28.png)

![Crisp image](https://i.imgur.com/401fBdJ.png)

**Comment by Jezzamonn** [comment]
Right, yeah I see what you mean. I did actually pick the smooth images _kind of_ on purpose just to make them look smooth to emphasise the fact they're waves, but it might be better to do make them pixelated. The cat image below is pixelated to emphasise the separate blocks.

Let me think and get back to this!
